.github/workflows/create-release-candidate.yml

```diff
@@ -1,12 +1,13 @@
-name: Create Release Candidate (6.x)
+name: Create Release Candidate
```

Comment:
```suggestion
name: Create Release
```

and also rename this file to `create-release.yml`?

Reply:
Yep! Why not